app/views/users/sessions.py

```diff
@@ -141,6 +146,73 @@ def add_speaker_view(event_id, session_id):
         return redirect(url_for('.index_view', event_id=event_id))
 
 
+@event_sessions.route('/download_speakers_as_csv/')
+@belongs_to_event
+@can_access
+def download_speakers_as_csv(event_id):
+    speakers = DataGetter.get_speakers(event_id)
+    event = DataGetter.get_event(event_id)
+    main = [["Speaker Name", "Speaker Email", "Speaker Session(s)", "Speaker Status", \
+             "Speaker Mobile", "Speaker Organisation", "Speaker Position"]]
+    for speaker in speakers:
+        data = [speaker.name if speaker.name else '', speaker.email if speaker.email else '']
+        if speaker.sessions:
+            session_names = ''
+            session_status = ''
+            for sess in speaker.sessions:
```

Comment:
Please name the variables properly. `sess` doesn't make any sense. Call it `session`